iis-common/src/main/java/eu/dnetlib/iis/common/string/StringUtils.java

```diff
@@ -0,0 +1,25 @@
+package eu.dnetlib.iis.common.string;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Common string utils.
+ */
+public class StringUtils {
+
+    /**
+     * Decodes an UTF-8 encoded string.
+     *
+     * @param source UTF-8 encoded string.
+     * @return Decoded string.
+     */
+    public static String decodeFromUtf8(String source) {
```

Comment:
Not a fan of this function's name. It's actually [URI  percent-encoding](https://tools.ietf.org/html/rfc3986#section-2.1) that we are dealing with here.

Reply:
It caught my eye as well. Maybe `decodeURL()` would be better here? Or something else describing the context more precisely.

Reply:
Yes, the name was chosen without some more thought. I've removed the whole class and added a resource handling class `eu.dnetlib.iis.common.utils.ResourceUtils` with a dedicated method `resourcePath(ClassLoader cl, String name)` for extracting resource path with proper formatting.